README.md

```diff
@@ -1 +1,69 @@
-# awesome-books
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Awesome books
+
+> This is a library project with a great UX and a beautiful UI.
+
+![screenshot](./screenshot.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- HTML
+- CSS
+- Javascript
+- Technologies used
+
+## Getting Started
+
+This project can be used to store books in the online library. Users can enter the title and the author of a book. Also, it allows users to remove books from the library.
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
+## Authors
+
+👤 **Abror Mukimov**
+- GitHub: [@abrormukimov](https://github.com/abrormukimov)
+- Twitter: [@abrormukimov](https://twitter.com/abrormukimov)
+- LinkedIn: [AbrorMukimov](https://www.linkedin.com/in/abror-mukimov/)
+
+👤 **Ritta Buyaki**
+- Github: [@Buyaki01](https://github.com/Buyaki01)
+- Twitter: [ @BuyakiRitta](https://twitter.com/BuyakiRitta)
+- Linkedin: [Ritta Buyaki](https://www.linkedin.com/in/ritta-buyaki-b12904128/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Buyaki01/awesome-books/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Please, it would be better if you add the MIT license file. The MIT link is broken because It is missing one. The README file is one of the most important things in a project because It helps other contributors, people interested in your project to understand how your project works or how to run it in a local environment. 😉